packages/web-components/@sfx/sayt/package.json

```diff
@@ -19,9 +19,11 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "@types/debounce": "^1.2.0",
```

Comment:
I don't remember if I recommended that this should be here, but since we'll exporting a type that requires the type of `debounce` (if you implement my suggested type), this has to be listed under `dependencies`. This [StackOverflow post](https://stackoverflow.com/a/46011417) has more detail.